sgt-back-end/index.js

```diff
@@ -0,0 +1,128 @@
+const express = require('express');
+const pg = require('pg');
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+    select *
+    from grades;
+  `;
+
+  db.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      res.status(200).json(grades);
+    })
+    // eslint-disable-next-line node/handle-callback-err
+    .catch(error => {
+      res.status(500).json({ error: 'An unexpected error occurred' });
+    });
+
+});
+
+app.post('/api/grades', (req, res) => {
+  const { name, course, score } = req.body;
+
+  const sql = `
+    insert into "grades" ("nme","course","score")
+    values ('${name}','${course}','${score}')
+    returning *;
+  `;
+  if (!name || !course || !score || score < 0 || score > 100) {
```

Comment:
This won't catch score if it is a decimal or a string of characters. You need to convert `score` to a number by doing something like `score = Number(score)` then validate is it an integer with something like `Number.isInteger(score)`.